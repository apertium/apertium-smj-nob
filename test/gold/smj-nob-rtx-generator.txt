[0zgkcjgRm7pk]
Du ser et brev. [/option]
[/0zgkcjgRm7pk]
[uksRfzth6VJb]
Jeg ser et brev. [/option]
[/uksRfzth6VJb]
