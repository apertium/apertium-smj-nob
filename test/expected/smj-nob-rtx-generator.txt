[--Jf4vZvr_ye#0] *Såj *goappátjagá *libá en glede *ja *mannaba en stue, *la *muddo *juojddá *bårråt *ja *vehi *tjátjev *juhkat!
[/--Jf4vZvr_ye]
[0_4JAoG6L__L#0] *Marit *tjajmmá *ja *javllá: "*Gávnniv *duv!".
[/0_4JAoG6L__L]
[0zgkcjgRm7pk#0] det ser et brev.
[/0zgkcjgRm7pk]
[1RrnT9EuJHGn#0] *Marit *gæhttjá en mor glass *duogen, en mor *tjajmmá.
[/1RrnT9EuJHGn]
[23xEDFbmNagz#0] *Ij *la en fugl *ij *makkirak *ietjá *judos.
[/23xEDFbmNagz]
[3gsPqKKORKL5#0] Dere ser et brev.
[/3gsPqKKORKL5]
[3j8W86jVUsTv#0] *De *sån det ser en #liten arm *aj *ja *gå *sån *lagábuj *boahtá, *de *sån #se *suv en sjef *aj!
[/3j8W86jVUsTv]
[4Y-4NSaqg2lP#0] *De *sån *gullá en stemme, *dat *boahtá *stuora #gammel trær *duogen.
[/4Y-4NSaqg2lP]
[4Y-bseVjsUxi#0] *Duommá *ja *Marit *lijkkuba #leke, *såj en alder *ståhkaba *aktan et gjerde *stuora stuer *ålggolin.
[/4Y-bseVjsUxi]
[5CZh_0gPiKjg#0] *Sån *gatját en: "hund #Se *gus *Duomisav?".
[/5CZh_0gPiKjg]
[6IdpD1gy9t4S#0] *Manen *dahká *nav? *Ja *majt *Duommá *dahká trær *guoran?
[/6IdpD1gy9t4S]
[7qwZfZdhM64z#0] *Vuojnáv *stuora brev *ruoppsis boliger *duogen.
[/7qwZfZdhM64z]
[8SBFlK0nZxXJ#0] *Sån *åhtså *Duommáv: *gåsi *sån *manáj? #Se *gus *suv?
[/8SBFlK0nZxXJ]
[C8vim-nGYG-S#0] *Vuojnáv *stuora brev boliger *duogen.
[/C8vim-nGYG-S]
[CEH5yciMA24N#0] *Le *gus *Maritin *aj en hund? *Ij, *Maritin *ij *la en hund, *sujna en l #forsvinne.
[/CEH5yciMA24N]
[Cjwibi5f_oK8#0] *Hæhttu *liehket *Duommá!
[/Cjwibi5f_oK8]
[DLpXFTApVfn7#0] *Marit #gå *hihtásit gjerder *tjadá.
[/DLpXFTApVfn7]
[Ggp9P9c68DSX#0] *Le *buorre et vær *uddni, *la *huj #varm.
[/Ggp9P9c68DSX]
[Hww_bn3eeY_i#0] *Duommá *jåhtelit *viehká *stuora #gammel trær *lusi, *sån et gjemmested *Maritis.
[/Hww_bn3eeY_i]
[I2x9TEKAgNs5#0] *Diedá *gus *manen? *Marit en topp *ja *sujna *li armer øyne *åvdån.
[/I2x9TEKAgNs5]
[IfiQnL4HXxRd#0] *Sån *gæhttjá *juohkka et sted, *valla *ij #kunne *Duomisav *gávnnat.
[/IfiQnL4HXxRd]
[Kl5Vm5-ZQnOK#0] *Valla en hund en viten *ij #kunne #tale! *Marit *ij *oattjo et svar et spørsmål.
[/Kl5Vm5-ZQnOK]
[Ku-nHnEt7guK#0] *Duommá *ja *Marit *libá et gjerde.
[/Ku-nHnEt7guK]
[LEyoga2qKOGR#0] *Valla en katt en, stue #forsvinne *oadá.
[/LEyoga2qKOGR]
[NhmjSU_bN_iB#0] *Dat *la et spill.
[/NhmjSU_bN_iB]
[SW-nn0XY3egZ#0] "#IV *Marit, #IV #kunne *subtsastit *dunji!", *sån *vásstet.
[/SW-nn0XY3egZ]
[TvvJw8-jH-6E#0] *Vuojnáv *stuora brev.
[/TvvJw8-jH-6E]
[Tz6hmt3JQRnA#0] *Vuojnáv *stuora et brev.
[/Tz6hmt3JQRnA]
[VizkGffGIbXD#0] *Marit *ij #vite *gånnå *Duommá en l
[/VizkGffGIbXD]
[WfnIzto84EKX#0] *Aktak *ij *oattjo et svar, *gå hunder *hålli!
[/WfnIzto84EKX]
[XObHLCGjyAD_#0] *Ij *majdik #se *ja *sån *javllá.
[/XObHLCGjyAD_]
[_YcYE3WfKa8F#0] *Marit *ájádallá, *jut *sån *la #se *gåsi *Duommá en l #reise: "*Subtsasta *munji *gånnå *sån *la!", *sån *javllá en mor.
[/_YcYE3WfKa8F]
[dBowB32b0DGA#0] *Duommá en l #liten *báhtjasj *ja *sån *la *gudá *jagák.
[/dBowB32b0DGA]
[dIEbIp9Wihqj#0] *Gå *Marit *gærggá #telle, *sån *gæhttjá *ietjas *birra.
[/dIEbIp9Wihqj]
[dl3AmGMpWzb9#0] Vi ser et brev.
[/dl3AmGMpWzb9]
[gPrvAQE1srBv#0] *Vájku *sån *ihkap *diehtá *gånnå *báhttja *la, *ij #ville *subtsastit.
[/gPrvAQE1srBv]
[h3-9CIErTsIn#0] *Sunnu en mor en stue #forsvinne *siegen, *sån *gæhttjá *vinndegav *ålgus *ja #se *Duommá *ja *Marit *libá #leke.
[/h3-9CIErTsIn]
[jRBe3Ks27s25#0] *Valla *iektu *lij *huj #kjølig! *Ejga #kunne #leke *ålggon *dalloj.
[/jRBe3Ks27s25]
[k7S6wvR1tb5e#0] *Sån *gæhttjá bord *vuolen *ja stoler *vuolen, *valla *Duommá *ij *la *dieppe.
[/k7S6wvR1tb5e]
[kwKFaodsNogv#0] *Sån *ájn *gæhttjalit *Duomisav #leite.
[/kwKFaodsNogv]
[nk32D9V8gQAw#0] *Dálla *sån *gullá *dav *buoragit.
[/nk32D9V8gQAw]
[qgyd6qGiVd0z#0] De ser et brev.
[/qgyd6qGiVd0z]
[tZoJWmCd_1o_#0] en Hund #like #leke *dajna *guovtijn barn.
[/tZoJWmCd_1o_]
[tiz0Jo2iJfCv#0] en #Liten småjente *suv en lillesøster, *sån *la *vidá *jagák.
[/tiz0Jo2iJfCv]
[uksRfzth6VJb#0] *Vuojnáv et brev.
[/uksRfzth6VJb]
[ut4tXmXaOnQo#0] *Máhtáluluj *gus *liehket *Duommá? *Dannák *dat en stemme *vas! *Sån *dárkkelit *gulldalit.
[/ut4tXmXaOnQo]
[yCmzXzgXooq1#0] en Hund *huj en glede *dálla.
[/yCmzXzgXooq1]
[yLLO_Y8NOsTl#0] *GÅNNÅ #L *DUOMMÁ?
[/yLLO_Y8NOsTl]
[zfeB5hXhzgNc#0] *Duomisin *la en #liten hund, *dat *la *aj et gjerde *dálla.
[/zfeB5hXhzgNc]
